src/cluecode/finder.py

```diff
@@ -59,6 +59,23 @@ def logger_debug(*args):
 """
 
 
+def check_if_line_can_contain_urls(line):
+    """
+    Returns true if the line is a likely candidate to include urls.
```

Comment:
Use imperative style: Return and not Returns

Reply:
Addressed your comment. :)